spec/models/claim/base_claim_spec.rb

```diff
@@ -70,7 +70,7 @@ class MockSteppableClaim < Claim::BaseClaim
   end
 
   describe '.agfs_claim_types' do
-    specify { expect(described_class.agfs_claim_types).to match_array([Claim::AdvocateClaim, Claim::AdvocateInterimClaim, Claim::AdvocateSupplementaryClaim]) }
+    specify { expect(described_class.agfs_claim_types).to match_array([Claim::AdvocateClaim, Claim::AdvocateInterimClaim, Claim::AdvocateSupplementaryClaim, Claim::AdvocateHardshipClaim]) }
```

Comment:
Ditto helper method

Reply:
this was slightly different from others (testing objects rather than string representation of object) so have made it more akin to similar tests and used the helpers